src/commands/plugins.rs

```diff
@@ -52,7 +52,7 @@ impl PluginCommands {
 }
 
 /// Install plugins from remote source
-#[derive(Parser, Debug)]
+#[derive(Parser, Debug, Default)]
```

Comment:
The default would be invalid.  If we want to avoid noise at the construction site, maybe add a `from_name` constructor rather than implying it can be defaulted?

Reply:
Are you saying the default is invalid because there is not name? I wanted a way to construct the `Install` struct:
```
                let plugin_installer = Install {
                    name: Some("cloud".to_string()),
                    yes_to_all: true,
                    ..Default::default()
                };
```
https://github.com/fermyon/spin/blob/7f63517f2361521cc02a6273d6c1c8815393b516/src/commands/external.rs#L55

Reply:
im being picky, ill just set the rest directly to none